### DISCUSSION TITLE
Lab Assignment #3 👩‍💻

### DISCUSSION THREAD

**Discussion by albertkun** [albertkun/23S-ASIAAM-191A#13] [Assignments]
## Due 4/27 before class 

Please use this thread to post your [lab assignment's #3](https://albertkun.github.io/23S-ASIAAM-191A/assignments/week3/lab_assignment/) link, which is the GitHub pages URL. [Click here for a reminder on how to get it](https://albertkun.github.io/23S-ASIAAM-191A/labs/week1/3/#using-github-pages).

### Note 
If you are submitting in two parts, put the part in the link you are submitting

#### For example:
Part1 - https://albertkun.github.io/23s-asiaam-191a-sample/week3a/index.html
Part2 - https://albertkun.github.io/23s-asiaam-191a-sample/week3b/index.html

If you only have one page, then you can just submit the `index.html` link as per usual.

**Comment by czhangy** [comment]
https://czhangy.github.io/ASIA-AM-191A/week3/maps.html

**Comment by SageLuong** [comment]
https://sageluong.github.io/23s-asianam-191a-labs/week3a/index.html
https://sageluong.github.io/23s-asianam-191a-labs/week3b/index.html

**Comment by mandymwlai** [comment]
https://mandymwlai.github.io/AA-191-lab/week3A/

**Comment by paige-lee** [comment]
https://paige-lee.github.io/aa191a-labs/week3a/index.html
https://paige-lee.github.io/aa191a-labs/week3b/index.html

**Comment by zacharypakin** [comment]
https://zacharypakin.github.io/aa191a-23s/week03/index.html

**Comment by ariscorpus** [comment]
https://ariscorpus.github.io/23s-asianam-191a-labs/week3a/
https://ariscorpus.github.io/23s-asianam-191a-labs/week3b/

**Comment by kmtsujimoto** [comment]
https://kmtsujimoto.github.io/23s-asiaam-191a-labs/week03/index.html

**Comment by joanneluong** [comment]
https://joanneluong.github.io/aas191a-labs/week3/part1.html

**Comment by maizaha** [comment]
https://maizaha.github.io/23s-asiaam-191a-labs/week3a/index.html
https://maizaha.github.io/23s-asiaam-191a-labs/week3b/index.html

**Comment by catherinehhu** [comment]
https://catherinehhu.github.io/191a-labs/week3a/index.html
https://catherinehhu.github.io/191a-labs/week3b/map.html

**Comment by vickydelk** [comment]
https://vickydelk.github.io/aa191-assignments/week3/index.html

**Comment by jamesfalkenroth** [comment]
https://jamesfalkenroth.github.io/23s-asianamerican-191a-labs/week3a/index.html
https://jamesfalkenroth.github.io/23s-asianamerican-191a-labs/week3b/index.html

**Comment by shintuzd** [comment]
https://shintuzd.github.io/23s-asiaam-191a-assignments/week3/maps.html

**Comment by emai14** [comment]
Part 1: https://emai14.github.io/23S-ASAM191A-Labs/week03A/
Part 2: https://emai14.github.io/23S-ASAM191A-Labs/week03B/

**Comment by angelicaalacantar** [comment]
https://angelicaalacantar.github.io/aas191a-23s/week3a/index.html
https://angelicaalacantar.github.io/aas191a-23s/week3b/index.html

**Comment by cassiuscarandang** [comment]
https://cassiuscarandang.github.io/23s-asiaam-191a-lab1/week3a/index.html
https://cassiuscarandang.github.io/23s-asiaam-191a-lab1/week3b/index.html

**Comment by cassidytu** [comment]
https://cassidytu.github.io/asam191a-labs/week3/map.html

**Comment by qatle** [comment]
https://trinhqale.github.io/AM191A-labs/week3/index.html

**Comment by alanamalika** [comment]
https://alanamalika.github.io/23s-asianam-191a-labs/week03/index.html
https://alanamalika.github.io/23s-asianam-191a-labs/week03/part1.html

**Comment by DylanJWan** [comment]
https://dylanjwan.github.io/asiaam-191a-assignments/portfoliosite/maps

**Comment by enyi-emuka** [comment]
https://enyi-emuka.github.io/asiaam191a_labs/week3b/index.html

**Comment by danigold1020** [comment]
https://danigold1020.github.io/23s-asianam-191a-labs/week03b/index.html

**Comment by trufflelover** [comment]
https://trufflelover.github.io/23s-asiaam-191a-labs/week3/FoodMap.html

**Comment by chrysanthemma** [comment]
https://chrysanthemma.github.io/asainam191a-labs/week1/index.html
